libs/angular-components/community/components/form/feedback-text/feedback-text.component.scss

```diff
@@ -0,0 +1,33 @@
+@use "@tehik-ee/tedi-core/mixins";
+
+.tedi-feedback-text {
+  --_font-weight: var(--font-weight-regular);
+  --_color: var(--general-text-secondary);
+
+  @include mixins.responsive-styles(
+    --_line-height,
+    body-small-regular-line-height
+  );
+  @include mixins.responsive-styles(--_font-size, body-small-regular-size);
+  @include mixins.responsive-styles(
+    --_font-family,
```

Comment:
elsewhere haven't used font-family as a local variable, don't think it is necessary here either